pkg/api/imagebasedgroupupgrades/v1alpha1/types.go

```diff
@@ -0,0 +1,97 @@
+package v1alpha1
+
+import (
+	lcav1alpha1 "github.com/openshift-kni/lifecycle-agent/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:generate=true
+
+// ImageBasedUpgradeAction defines the type for the actions field
+// +kubebuilder:validation:Enum=Abort;Prep;Upgrade;Rollback;Finalize
+type ImageBasedUpgradeAction string
```

Comment:
Safer to make this a struc for future extension

Reply:
updated